monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -4470,6 +4470,23 @@ object Observable {
     override def empty[A]: Observable[A] =
       Observable.empty[A]
   }
+
+  /** [[cats.NonEmptyParallel]] instance for [[Observable]]. */
+  implicit def observableNonEmptyParallel: NonEmptyParallel[Observable, CombineObservable.Type] =
+    new NonEmptyParallel[Observable, CombineObservable.Type] {
+      import CombineObservable.unwrap
+      import CombineObservable.{apply => wrap}
+
+      def flatMap: FlatMap[Observable] = implicitly[FlatMap[Observable]]
```

Comment:
Use `override` even on abstract methods, because we need to protect against API breakage.